src/components/calcite-scrim/calcite-scrim.scss

```diff
@@ -27,7 +27,7 @@
   items-center
   flex-col
   justify-center;
-  animation: calcite-scrim-fade-in 250ms ease-in-out;
+  animation: calcite-scrim-fade-in calc(250ms * var(--calcite-animation-timing-factor)) ease-in-out;
```

Comment:
Since there's only a 50ms difference between the internal global vars, could this use either the `medium` or `slow` var? cc @macandcheese

Reply:
Yeah let's standardize any outliers... would suggest slow since scrim can cover a large visual area but happy to adjust after we standardize if needed.